src/corehost/cli/deps_entry.cpp

```diff
@@ -167,19 +169,21 @@ bool deps_entry_t::to_hash_matched_path(const pal::string_t& base, pal::string_t
 
     // Build the nupkg file name. Just reserve approx 8 char_t's for the algorithm name.
     pal::string_t nupkg_filename;
-    nupkg_filename.reserve(library_name.length() + 1 + library_version.length() + 16);
-    nupkg_filename.append(library_name);
+    pal::string_t lower_name = pal::to_lower(library_name);
+    pal::string_t lower_version = pal::to_lower(library_version);
+    nupkg_filename.reserve(lower_name.length() + 1 + lower_version.length() + 16);
+    nupkg_filename.append(lower_name.c_str());
     nupkg_filename.append(_X("."));
-    nupkg_filename.append(library_version);
+    nupkg_filename.append(lower_version.c_str());
     nupkg_filename.append(_X(".nupkg."));
     nupkg_filename.append(library_hash.substr(0, pos));
 
     // Build the hash file path str.
     pal::string_t hash_file;
-    hash_file.reserve(base.length() + library_name.length() + library_version.length() + nupkg_filename.length() + 3);
+    hash_file.reserve(base.length() + lower_name.length() + lower_version.length() + nupkg_filename.length() + 3);
     hash_file.assign(base);
-    append_path(&hash_file, library_name.c_str());
-    append_path(&hash_file, library_version.c_str());
+    append_path(&hash_file, lower_name.c_str());
+    append_path(&hash_file, lower_version.c_str());
     append_path(&hash_file, nupkg_filename.c_str());
 
```

Comment:
Sorry, just saw this is on the releas branch. The mobile UI didn't show the branch. @gkhanna79 this is tested with CLI's CI quite a while back. But did not make the RC2 checkin.